### DISCUSSION TITLE
Overriding Sonarr Metadata

### DISCUSSION THREAD

**Discussion by zpz5HAU-tgc3fgw2xwr** [meisnate12/Plex-Meta-Manager#192] [Ideas]
My current setup uses Sonarr to automate renaming of tv shows and manage the collection. The issue I'm having is mostly with TheTVDB being Sonarr's only source of truth. The problems I'm having because of that:

- Lots of series are completely locked from add/changing episodes
- Specials episode ordering is locked for all series
- I don't want to visit their site to make changes every day

Your project solves this issue on Plex's side, allowing me to reliably save my changes locally. But now (mainly for ongoing web series) I'm noticing an issue on Sonarr's end. If I rely on Plex-Meta-Manager and my YML config instead of manually entering  and waiting for changes to TheTVDB, Sonarr won't recognize those changes. The file names will be incorrect if ordering needs to change, and it also won't be aware of new episodes.

This might be out of the scope for this project, but if we had a way to override Sonarr's metadata for episodes that would solve the problem.

**Comment by meisnate12** [comment]
sorry i never noticed this question before but unfortunately, you're right that ask would be out of scope for PMM
